### PR TITLE
Expose Prometheus endpoint for Hawkular Services uses

### PR DESCRIPTION
**Pull request by lucasponce**
Minor one, as we might want to reuse same property for Hawkular Services and then it makes sense to not make it dependent on alerting.

### CODE REVIEW

**Comment by lucasponce** [comment]
DNM label.
I'm not fully convinced of this PR I'm going to change it a little bit.

File: hawkular-alerters/hawkular-prometheus-alerter/deployments/hawkular-prometheus-alerter-standalone/src/main/webapp/WEB-INF/jboss-web.xml [hunk 1]
```diff
@@ -18,5 +18,5 @@
 
 -->
 <jboss-web>
-  <context-root>/hawkular/__alerter_prometheus</context-root>
+  <context-root>/hawkular/alerter/prometheus</context-root>
```
Comment (lucasponce) [inline hawkular-alerters/hawkular-prometheus-alerter/deployments/hawkular-prometheus-alerter-standalone/src/main/webapp/WEB-INF/jboss-web.xml:21]: I unified the url with 2.x.
I guess there is no clash there and in this way, this endpoint are consistent with rest use cases.

**Comment by lucasponce** [comment]
Please @jshaughn take a look at this as I would need it to bump version in hServices.
Thx

**Comment by jshaughn** [comment]
Right, since we actually do expose an endpoint for notification consumption I think this makes sense.

**Pull request merged**